fetch.bs

```diff
@@ -3913,39 +3913,29 @@ steps:
     <a lt="CORS-preflight cache">cached</a>.
 
    <li>
-    <p>If all of the following are true
+    <p>If <var>request</var>'s <a for=request>method</a> is not in <var>methods</var> and
+    <var>request</var>'s <a for=request>method</a> is not a <a>CORS-safelisted method</a>, then:
 
-    <ul class=brief>
-     <li><var>request</var>'s <a for=request>method</a> is not in <var>methods</var>
-
-     <li><var>request</var>'s <a for=request>method</a> is not a <a>CORS-safelisted method</a>
-
-     <li><var>request</var>'s <a for=request>credentials mode</a> is not "<code>include</code>"
-     and <var>methods</var> does not contain `<code>*</code>`
-    </ul>
-
-    <p>then return a <a>network error</a>.
+    <ol>
+     <li><p>If <var>request</var>'s <a for=request>credentials mode</a> is "<code>include</code>" or
+     <var>methods</var> does not contain `<code>*</code>`, then return a <a>network error</a>.
+    </ol>
 
    <li><p>If one of <var>request</var>'s <a for=request>header list</a>'s
    <a lt=name for=header>names</a> is a <a>CORS non-wildcard request-header name</a> and is not a
    <a>byte-case-insensitive</a> match for an <a for=list>item</a> in <var>headerNames</var>, then
    return a <a>network error</a>.
 
    <li>
-    <p>If all of the following are true
+    <p>If one of <var>request</var>'s <a for=request>header list</a>'s <a for=header>names</a> is
```

Comment:
Isn't this confusing? Imagine _request_ has two headers, "Accept" and "foobar", and "foobar" is listed in _headerNames_. But this branch is taken wrongly because

 - "Accept" is not a byte-case-insensitive match for an item in _headerNames_, and
 - "foobar" is not a CORS-safelisted request header.

Reply:
Good catch, thanks! Posted a fix.